JS/Lesson25/gulcin/filter.js

```diff
@@ -182,6 +184,8 @@ const sentencesExercise5 = [
 	'The butterfly gracefully fluttered its wings, showcasing vibrant colors in a delicate dance.',
 ];
 
+const shortSentences = sentencesExercise5.map(sentence => sentence.split(' ')).filter(sentence => sentence.length < 10);
```

Comment:
Don't forget to join the words into sentences again :) 
```suggestion
const shortSentences = sentencesExercise5.map(sentence => sentence.split(' ')).filter(sentence => sentence.length < 10).map(sentence => sentence.join(' '));
```